wcfsetup/install/files/acp/templates/userList.tpl

```diff
@@ -56,7 +56,7 @@
 
 <header class="contentHeader">
 	<div class="contentHeaderTitle">
-		<h1 class="contentTitle">{lang}{@$pageTitle}{/lang}
+		<h1 class="contentTitle">{lang}{@$pageTitle}{/lang}</h1>
```

Comment:
Well, you fixed the markup, but the counter is still missing. Also, because it supports filtering, this list can be empty, requiring the counter to be hidden at times.